#!/usr/bin/python3
def raise_exception():
    x = "string"
    y = 5
    z = x + y
